src/04-calendar/events.apib

```diff
@@ -28,6 +28,7 @@ Get a list of calendar events.
                     + Members
                         + contact
                         + company
+                        + deal
```

Comment:
This exposes that we will start having `deal` type links, so we can add that entity type to the enum of `links` types on lines 78 and 122.

Reply:
We are only using it on the list. Does it make sense to add it to an end point if we are not using it. I assume this is a case of YAGNI You aren't going to need it.